src/EA.Weee.Web.Tests.Unit/Areas/Scheme/Controllers/ManageEvidenceNotesControllerTransferEvidenceNoteTests.cs

```diff
@@ -235,12 +265,49 @@ public async Task TransferEvidenceNotePost_GivenAModelIsValidAndRequestIsCreated
             //act
             var result = await ManageEvidenceController.TransferEvidenceNote(model) as RedirectToRouteResult;
 
-            /* to be updated */
+            // assert
+            /* to be updated when actual redirect happens */
             result.RouteValues["action"].Should().Be("Index");
             result.RouteValues["controller"].Should().Be("Holding");
             result.RouteValues["organisationId"].Should().Be(model.OrganisationId);
         }
 
+        [Fact]
+        public async Task TransferEvidenceNotePost_GivenRequestIsCreated_SessionShouldNotBeNull()
+        {
+            //arrange
+            var model = GetValidModel(Guid.NewGuid(), Guid.NewGuid());
+            var transferRequest = GetRequest();
+            var httpContext = new HttpContextMocker();
+            httpContext.AttachToController(ManageEvidenceController);
+
+            A.CallTo(() => TransferNoteRequestCreator.ViewModelToRequest(A<TransferEvidenceNoteDataViewModel>._)).Returns(transferRequest);
+
+            //act
+            await ManageEvidenceController.TransferEvidenceNote(model);
+
+            // assert
+            ManageEvidenceController.Session.Should().NotBeNull();
+            ManageEvidenceController.Session.SessionID.Should().BeNullOrEmpty();
```

Comment:
here is it possible to check that the Session["whatever the request session id is"].Should().NotBeNull()

i.e. that the created request is in the session

Reply:
okay.